libs/content/workspace/src/task_manager.rs

```diff
@@ -80,9 +87,6 @@ pub struct LoadRequest {
 #[derive(Clone)]
 pub struct SaveRequest {
     pub id: Uuid,
-    pub old_hmac: Option<DocumentHmac>,
-    pub seq: usize,
-    pub content: String,
```

Comment:
queue a save request with just the id; task manager will grab the rest when it's time to launch